### PR TITLE
LPS-97974,LPS-97975

### PR DESCRIPTION
**Pull request by danielkocsis**
Hey @Preston-Crary,

I put together the two LPS's related to remove versioning from Layouts and Layout sets. I added the basic upgrade process we discussed, however there was nothing related to 7.3 yet, so I am not 100% sure of this part. I run ci upgrade tests, there were no failures related to this one, but please review it.

The other thing is that we changed the counter service usage for them with introducing versioning to use entity specific counters instead of the generic one, I decided to keep it even if we rolling back versioning. 

Me and @giros are OoO for the rest of the week but please feel free to let us know if you think we it needs more changes.

thanks,
Daniel

### CODE REVIEW

**Comment by Preston-Crary** [comment]
ci:test:backend-hypersonic

**Comment by Preston-Crary** [comment]
Looks like there's some test compile failures when you guys get back can you check to see if there's any more of them like in LayoutSetPrototypePropagationTest?

**Comment by danielkocsis** [comment]
ci:test:backend-hypersonic

**Comment by danielkocsis** [comment]
I fixed it and pushed it again

File: modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/exportimport/data/handler/LayoutStagedModelDataHandler.java [hunk 1]
```diff
@@ -554,7 +554,9 @@ else if (layoutsImportMode.equals(
 		Layout importedLayout = null;
 
 		if (existingLayout == null) {
-			importedLayout = _layoutLocalService.create();
+			long plid = _counterLocalService.increment();
```
Comment (Preston-Crary) [inline modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/exportimport/data/handler/LayoutStagedModelDataHandler.java]: Can you do a search for users of the createLayout / createLayoutSet methods.  There's a couple more wrong counter names still being used.  Not sure if there's more.

**Comment by Preston-Crary** [comment]
There's also a lot of "draftLayout" results when searching.  Let's try to rename those variables too.

**Comment by danielkocsis** [comment]
Hey @Preston-Crary,

I replaced the counter call and searched for other occurrences but this was the only one.

Also checked for the draft usage, but it seems we had 2 kinds of drafts, at least the naming was ambiguous. WEM team introduced a versioning independent draft concept, a layout which has an attached model, another layout. I went trough all the occurrences, and it seems that now we only have the WEM draft in the code, not the SB one. I would not start to rename the whole thing now, what do you think?

Daniel

**Comment by Preston-Crary** [comment]
Please see https://github.com/shuyangzhou/liferay-portal/pull/8019

**Pull request closed without merging**